components/sensor/ld2410.rst

```diff
@@ -25,59 +25,261 @@ Use of hardware UART pins is highly recommended, in order to support the out-of-
 
     # Example configuration entry
     uart:
-      id: uart1
-      tx_pin: REPLACEME
-      rx_pin: REPLACEME
+      id: ld2410_uart
+      tx_pin: REPLACE ME
+      rx_pin: REPLACE ME
```

Comment:
These are `REPLACEME` (with no space) for a specific reason, because ESPHome actually looks for this exact value and give the user a config error

Reply:
done